### PR TITLE
fix(client): skip to content button overlapping the nav

### LINKED ISSUE #54194: The left side of the "Skip to content" button's outline is not visible
### Describe the Issue

It seems that the left side of the "Skip to content" button's outline is extending off the screen.

### Affected Page

Pages with `Header` component

### Steps to Reproduce

1. Go to [/learn](https://www.freecodecamp.org/learn) page
2. Press Tab key once to be focused on the "Skip to content" button and make it visible
3. See the outline of the button

### Expected behavior

The position of the button should be properly aligned and the outline should be visible.

### Screenshots

<details>
<summary>"Skip to content" button in /learn page</summary>

</details>

### System

- Device: Laptop
- OS: Windows 11 Home
- Browser: Chrome
- Version: 123.0.6312.58 (Official Build) (64-bit)

### Additional context

With reffering this [fCC's article](https://www.freecodecamp.org/news/how-to-add-skip-to-main-content-links-to-a-website/), the button and its outline should probably be aligned to the left edge of the search bar (on screens wider than or equal to 980px) or the fCC logo (on screens less than 980px).

### PR DESCRIPTION
**Pull request by Rabberpoli**
Checklist:
- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Updated 'left' position related to 'skip-to-content' button as stated in 54194 comments. Moreover I added a 'top' property in order to avoid button's outline to overlap freecodecamp header.

The result is as it follows:

width >= 980px:
Screenshot 2024-03-26 095318

width < 980px:
Screenshot 2024-03-26 095305

### CODE REVIEW

**Comment by Rabberpoli** [comment]
Sorry guys, I just realized that this issue has been solved by merging previous pull request (#54210) after submitting my pull request.

The only difference between my request and the one approved is that mine has slightly improved the button position by avoiding overlapping with main header; anyway in order to avoid code conflicts on merge let me know I if it's better to close this pull request. Thanks!

File: client/src/components/Header/header.css [hunk 1]
```diff
@@ -15,14 +15,16 @@
   left: -1000px;
 }
 
-@media screen and (min-width: 980px) {
```
[the remark this answers is not in the document]
Comment (Rabberpoli) [reply client/src/components/Header/header.css]: Just committed reverted changes on @media statement

File: client/src/components/Header/header.css [hunk 2]
```diff
@@ -15,14 +15,16 @@
...
+@media (min-width: 980px) {
   .skip-to-content-button:focus {
+    top: 50px;
     left: 15px;
   }
 }
 
-@media screen and (max-width: 979px) {
```
[the remark this answers is not in the document]
Comment (Rabberpoli) [reply client/src/components/Header/header.css]: Just committed reverted changes on @media statement

File: client/src/components/Header/header.css [hunk 3]
```diff
@@ -15,14 +15,16 @@
   left: -1000px;
 }
 
-@media screen and (min-width: 980px) {
+@media (min-width: 980px) {
   .skip-to-content-button:focus {
+    top: 50px;
```
[the remark this answers is not in the document]
Comment (Rabberpoli) [reply client/src/components/Header/header.css]: Sorry, I didn't see the `--header-height` variable. I added a fixed value of 10px both on desktop and mobile

**Review by huyenltnguyen — APPROVED** [review]

File: client/src/components/Header/header.css [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 .skip-to-content-button {
   position: absolute;
-  top: var(--header-height);
+  top: calc(var(--header-height) + 10px);
   border-radius: 15px;
```
Comment (ahmaxed) [inline client/src/components/Header/header.css:11]:
```suggestion
```

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**